portal-api/src/v1/validation/fields/cover-end-date.js

```diff
@@ -5,35 +5,49 @@ const {
   dateValidationText,
 } = require('./date');
 
-module.exports = (submittedValues, errorList) => {
-  const newErrorList = errorList;
+const { SUBMISSION_TYPE } = require('../../../constants/deal');
+const { DEAL_STATUS } = require('../../../constants/facilities');
+
+const isValidForValidation = (deal, submittedValues) => {
```

Comment:
should this read `isReadyForValidation` ?

Reply:
done